Src/BootCamp.Chapter/InputMonitor.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class InputMonitor : IInputMonitor<ConsoleKey>
+    {
+        public event EventHandler<InputPressedEventArgs<ConsoleKey>> InputPressed;
+
+        public void MonitorInput(ConsoleKey input)
+        {
+            OnInputPressed(input);
+            
+        }
+
+        protected virtual void OnInputPressed(ConsoleKey keyPressed)
```

Comment:
I think this function is not needed and you can call it in the function above direclty (withuot this function).